.github/workflows/test.yml

```diff
@@ -1,123 +1,68 @@
 name: Quality
-
 on:
   pull_request:
-  # schedule:
-    # Every morning at 6:00 AM CET
-    # - cron: '0 4 * * *'
   workflow_dispatch:
     inputs:
-      target-env:
-        description: 'Zitadel target environment to run the acceptance tests against.'
-        required: true
-        type: choice
-        options:
-          - 'qa'
-          - 'prod'
-
+      force:
```

Comment:
`force`? Shouldn't we use a more descriptive name like just `ignore-run-caches`?

Reply:
Good idea, i'll change that